db/migrate/20200511151348_add_automated_rsvp_to_workshop_invitation.rb

```diff
@@ -0,0 +1,5 @@
+class AddAutomatedRsvpToWorkshopInvitation < ActiveRecord::Migration
+  def change
+    add_column :workshop_invitations, :automated_rsvp, :boolean
```

Comment:
... several days later

Hi @despo

If there is any way it would help to have :automated_rsvp set to a boolean default at the database level? Currently, it is set to NULL/nil by the database by default and if the application code does not assign a boolean the nil will leak into the app. This is known as the three state boolean problem - booleans are only defined as true and false, the behaviour of a nil boolean .... well, is anyone's guess.

If you search the issues by whiny [something I wrote will pop to the top](https://github.com/codebar/planner/issues/1070#issuecomment-541415156) - I wrote about it in regard to `WorkshopInvitation[:attending]` but it applies to all the boolean data in the database including :automated_rsvp.

An example of the problems this leads to:
```
# inviation_concerns
scope :not_accepted, -> { where('attending is NULL or attending = false') }
                                               /\
                                               ||
   nuke it from orbit it's the only way to be sure 
```
[Link to code](https://github.com/codebar/planner/blob/master/app/models/concerns/invitation_concerns.rb#L15)

I believe the migration fix would look something like this (see references):
```
# a new migration
def up
    change_column_default :workshop_invitations, :automated_rsvp, false
end

def down
    change_column_default :workshop_invitations, :automated_rsvp, nil
end
```


### Reference
[Avoid the three state boolean problem](https://thoughtbot.com/blog/avoid-the-threestate-boolean-problem)
[Adding :default => true to boolean in existing Rails column](https://stackoverflow.com/questions/8627156/adding-default-true-to-boolean-in-existing-rails-column)
[Rails api - Change column default](https://api.rubyonrails.org/v5.1.7/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-change_column_default)